### PR TITLE
Add GET /api/node/info endpoint - Closes #5252

### LINKED ISSUE #5252: Add GET /api/node/info endpoint
### Description

Add the endpoints:

- `GET /api/node/info`

according to the following specification:

- they should return decoded node information obtained from the requested action

### Motivation

The HTTP API should return information about the attached node.

### Acceptance Criteria
- Update framework action `getNodeStatus` to `getNodeStatusAndConstants`
- The `getNodeStatusAndConstants` should respond with data according to definition https://github.com/LiskHQ/lisk-sdk/blob/0c2209c9f61fb96a3eaebcb782d3fb131ecf9c21/framework-plugins/lisk-framework-http-api-plugin/swagger.yml#L773 
- Add functional tests
- Register plugin to core, and check if it returns correct node information

### Additional Information
https://github.com/LiskHQ/lisk-sdk/blob/0c2209c9f61fb96a3eaebcb782d3fb131ecf9c21/framework/src/application/application.ts#L452
https://github.com/LiskHQ/lisk-sdk/blob/0c2209c9f61fb96a3eaebcb782d3fb131ecf9c21/framework/src/application/node/node.ts#L458
Related issues: #5213

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

This PR resolves #5252

### How was it solved?

- By adding new action `getNodeStatusAndConstants`
- By adding new controller for route /api/node/info

### How was it tested?

- build and run `node test/test_app/index.js`
- `curl http://localhost:4000/api/node/info | jq` (or use Postman/Browser with that URL)
- functional tests for new action and route

### CODE REVIEW

**Comment by pablitovicente** [comment]
@shuse2 should I remove `getNodeStatus` the one added here kind of replaces it though I was not sure if we want to keep it for other uses.

File: framework/src/application/application.ts [hunk 1]
```diff
@@ -470,6 +470,9 @@ export class Application {
 				getSchema: {
 					handler: () => this._node.actions.getSchema(),
 				},
+				getNodeStatusAndConstants: {
```
Comment (shuse2) [inline framework/src/application/application.ts:473]: `getNodeInfo` is fine?
Also, if the previous one is not used anymore, then lets remove it

File: framework-plugins/lisk-framework-http-api-plugin/src/controllers/node.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { Request, Response, NextFunction } from 'express';
+import { BaseChannel } from 'lisk-framework';
+
+export const getNodeStatusAndConstants = (channel: BaseChannel) => async (
+	_req: Request,
+	res: Response,
+	next: NextFunction,
+): Promise<void> => {
+	try {
+		const nodeStatusAndInfo: Buffer = await channel.invoke('app:getNodeStatusAndConstants');
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-http-api-plugin/src/controllers/node.ts:23]: bit confusing, `nodeStatusAndInfo` is json object? if its buffer then below should be decoded `send(nodeStatusAndInfo)`?

File: framework-plugins/lisk-framework-http-api-plugin/swagger.yml [hunk 3]
```diff
@@ -876,3 +876,11 @@ definitions:
                 type: integer
                 description: Distance in height between each milestone.
                 example: 3000000
+          communityIdentifier:
+            type: string
+            description: Community identifier the node is part of.
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-http-api-plugin/swagger.yml:881]:
```suggestion
            description: The unique name of the relevant community as a string encoded in UTF-8 format.
```

File: framework-plugins/lisk-framework-http-api-plugin/swagger.yml [hunk 4]
```diff
@@ -876,3 +876,11 @@ definitions:
...
+            example: Lisk
+          totalAmount:
+            type: string
+            description: Initial token senderPublicKey
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-http-api-plugin/swagger.yml:885]:
```suggestion
            description: Total amount of LSK available in network before rewards milestone started
```

File: framework/src/application/application.ts (hunk 1 above)
Comment (pablitovicente) [reply framework/src/application/application.ts]: Removed and Renamed.

File: framework-plugins/lisk-framework-http-api-plugin/src/controllers/node.ts (hunk 2 above)
Comment (pablitovicente) [reply framework-plugins/lisk-framework-http-api-plugin/src/controllers/node.ts]: Fixed

**Pull request merged**